cmd/load.go

```diff
@@ -355,33 +354,6 @@ func (p *LoadParams) configureNSCEnv() error {
 	return nil
 }
 
-func (p *LoadParams) configureNATSCLI() error {
```

Comment:
for background, I want this gone because it's causing some weird dependency issues with new `nats auth` behaviours - so @wallyqs what was this for and I can add similar functionality to `nats auth`?

Also we cant call Select here to change peoples selected contexts its just rude and unsafe, so for now just remove the whole lot